app-admin/yadm/yadm-2.4.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1
+
+DESCRIPTION="A dotfile manager for the config files in your home folder"
+HOMEPAGE="https://github.com/TheLocehiliosan/yadm/"
+SRC_URI="https://github.com/TheLocehiliosan/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	test? (
+		  dev-python/pytest
+		  dev-python/pylint
+		  || ( dev-util/shellcheck dev-util/shellcheck-bin )
+		  dev-python/flake8
+		  dev-vcs/git
```

Comment:
as for deps, we typically try to avoid linting tests (flake8 pylint), all they do is print warnings for lints, not testing real functionality.

Reply:
This is fair enough